spec/components/trainee_status_card/view_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe TraineeStatusCard::View do
```

Comment:
Do we need a view preview for `TraineeStatusCard`?

Reply:
I wasn't sure, TraineeStatusCard relies on the html/css in StatusCard, so I guess it would be the same preview visually for both? So I just decided to preview StatusCard

Reply:
It's the `TraineeStatusCard` that does all the mapping of states to colours and counts, so it might be nice for us to have a preview of these:

You could whip them up quickly with:

```
    Trainee.states.keys.each do |state|
      define_method state.to_s do
        render(TraineeStatusCard::View.new(state: state, target: trainees_path("state[]": state), trainees: @trainees))
      end
    end
```

Reply:
I've pushed up the preview :)

Reply:
i'm so sorry about the word 'quickly'. it was actually trickier than i expected!